src/ui-lib/sass/05-components/03-organisms/00-page-structure/_01-page-footer.scss

```diff
@@ -7,7 +7,6 @@ body > footer {
 
 
   > * {
-    @include grav-l-container;
```

Comment:
We shouldn't be removing this. This is what ensures that footer content does not exceed the max content with that the rest of the page has.